base_search_fuzzy/models/ir_model.py

```diff
@@ -43,7 +43,7 @@ def decorate_leaf_to_sql(self, eleaf):
             if left in model._fields:
                 params = str(right)
 
-            if isinstance(params, basestring):
+            if isinstance(params, pycompat.string_types):
```

Comment:
Is this something made by `2to3`? I think `isinstance(params, str)` if better

Reply:
Indeed, there's no need to support py2 for v11.